### PR TITLE
Add linting to webpack-sources.

### PR DESCRIPTION
**Pull request by chriseppstein**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [x] Increase the version number in the header if appropriate.
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

This patch does not change the API, it only adds linting support and the changes required to make the linter pass.

### CODE REVIEW

**Comment by chriseppstein** [comment]
I have some other API compat changes to make but I wanted to get the linter in place first.

File: types/webpack-sources/index.d.ts [hunk 1]
```diff
@@ -1,15 +1,14 @@
-// Type definitions for webpack-sources v0.1.2
+// Type definitions for webpack-sources 0.1
```
Comment (e-cloud) [inline types/webpack-sources/index.d.ts]: I haven't kept up with DefinitelyTyped's update, Is this mean we don't need `v` prefix?

Comment (chriseppstein) [inline types/webpack-sources/index.d.ts]: @e-cloud Right, the linter specifically forbids it now.

**Comment by e-cloud** [comment]
:+1:

**Pull request merged**